### PR TITLE
Updated docs for EndpointSliceTerminatingCondition

### PR DESCRIPTION
**Pull request by YashPimple**
Resolved Issue #38387 Stale docs for EndpointSliceTerminatingCondition 

Made a new PR, similar to #37913, but targeting the main branch as per Instructed.

### CODE REVIEW

**Comment by mehabhalodiya** [comment]
Thanks @YashPimple 

/lgtm

File: content/en/docs/concepts/services-networking/endpoint-slices.md [hunk 1]
```diff
@@ -87,16 +87,16 @@ The three conditions are `ready`, `serving`, and `terminating`.
 
 #### Ready
 
-`ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
-condition set to `True` should have this EndpointSlice condition also set to `true`. For
+`ready` is a condition that maps to a Pod's `ready` condition. A running Pod with the `ready`
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/endpoint-slices.md:90]: No. Pod's condition is called `Ready`, not `ready`.

Comment (mehabhalodiya) [reply content/en/docs/concepts/services-networking/endpoint-slices.md:90]: @YashPimple Please do the changes according to the suggestion.

**Comment by sftim** [comment]
/lgtm cancel

We should not use the wrong condition name.

File: content/en/docs/concepts/services-networking/endpoint-slices.md [hunk 2]
```diff
@@ -87,18 +87,18 @@ The three conditions are `ready`, `serving`, and `terminating`.
 
 #### Ready
 
-`ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
-condition set to `True` should have this EndpointSlice condition also set to `true`. For
-compatibility reasons, `ready` is NEVER `true` when a Pod is terminating. Consumers should refer
+`Ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/endpoint-slices.md:90]: Checked the source code. For Endpoints, the unofficial condition type is `ready`, for Pod, the official condition type is `Ready`. This inconsistency is annoying, but it is the fact to be documented.

Comment (sftim) [reply content/en/docs/concepts/services-networking/endpoint-slices.md:90]: > unofficial

But part of Kubernetes, right?

We might want to (separately) sprinkle some note shortcodes to point out the oddness.

**Review by sftim — COMMENTED** [review]
From elsewhere in the file:
> ```markdown
> `Terminating` is a condition that indicates whether an endpoint is terminating.
> ```

Is that actually `terminating`?

**Comment by mehabhalodiya** [comment]
👋🏻 @YashPimple Please do the changes suggested above and rebase your PR too.

Here are some references/tips:

- [merge conflicts and rebasing](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#merge-conflicts-and-rebasing)
- [squashing commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits)

Another option would be:
Close this pull request, and open a new one based on the main branch that makes the same changes. In the new PR, mention that it's a replacement for PR https://github.com/kubernetes/website/pull/38390

Thank you.

**Comment by sftim** [comment]
Replaced by https://github.com/kubernetes/website/pull/38701

**Pull request closed without merging**